tests/cypress/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
     "extends": "./../../tsconfig.base.json",
     "compilerOptions": {
-        "baseUrl": "./../..",
+        "baseUrl": ".",
```

Comment:
This was causing warnings in the IDE -- now pointing to the correct base folder to resolve the imports location